draft.md

```diff
@@ -727,7 +749,7 @@ polymorphic_value& operator=(const polymorphic_value& pv);
 
 * _Returns_: `*this`.
 
-* _Ensures_:  The state of `*this` is as if copy constructed from `pv`.
+* _Postconditions_:  The state of `*this` is as if copy constructed from `pv`.
```

Comment:
A little unclear on Daniel Krügler's comments here:

> [polymorphic_value.assignment], move assignment operator: The function is specified to be noexcept but has the following Effects: Equivalent to polymorphic_value(pv).swap(*this). These Effects say that it would call the *copy* constructor (which may throw). I assume that the intention is to call the *move* constructor, but that would require to reword the Effects to say instead: Equivalent to polymorphic_value(std::move(pv)).swap(*this).

This appears as though the copy constructor comments have been applied to the move constructor???